telluric/georaster.py

```diff
@@ -1076,9 +1089,17 @@ def bounds(self):
 
     def footprint(self):
         """Return rectangle in world coordinates, as GeoVector."""
+        if self._footprint is not None:
+            return self._footprint
         corners = [self.corner(corner) for corner in self.corner_types()]
-        shp = Polygon([[corner.get_shape(corner.crs).x, corner.get_shape(corner.crs).y] for corner in corners])
-        return GeoVector(shp, self.crs)
+        coords = []
+        for corner in corners:
+            shape = corner.get_shape(corner.crs)
+            coords.append([shape.x, shape.y])
+
+        shp = Polygon(coords)
+        self._footprint = GeoVector(shp, self.crs)
```

Comment:
What about using `GeoVector.from_bounds`?

Reply:
I just copied that was there in the first place, the only thing I added is just caching the product of the calculation 

so the question is to you, I guess

Reply:
Fair enough, could you please add a `# TODO` here so I don't forget? I'll do it after we merge this pull request.

Reply:
it's about time :)